### PR TITLE
Initialization of the repository with pom.xml 

### PR DESCRIPTION
**Pull request by dimuthnc**
## Purpose
pom.xml and  .gitignore files added.

### CODE REVIEW

**Comment by dimuthnc** [comment]
@pulasthi7 @madawas

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  -->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.wso2</groupId>
+        <artifactId>wso2</artifactId>
+        <version>1</version>
+    </parent>
+    <groupId>org.wso2.carbon</groupId>
```
Comment (pulasthi7) [inline pom.xml]: this should be "org.wso2.carbon.identity.conditional.auth.functions"

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    <artifactId>org.wso2.carbon.conditional.auth.functions</artifactId>
```
Comment (pulasthi7) [inline pom.xml]: This should be "identity-conditional-auth-functions"

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    <version>0.1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>WSO2 Carbon - IAM Conditional Authentication Functions</name>
+    <description>
+    </description>
+    <url>http://wso2.org</url>
```
Comment (pulasthi7) [inline pom.xml]: Add SCM tag also, please check some other extension repo root pom for any such missing tags.

**Pull request merged**